packages/react/src/components/CTA/CTA.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { ArrowDown20, ArrowRight20, Launch20 } from '@carbon/icons-react';
+import { ButtonGroup } from '../../patterns/sub-patterns/ButtonGroup';
+import { CardLink } from '../../patterns/sub-patterns/CardLink';
+import { FeaturedLink } from '../../patterns/blocks/FeaturedLink';
+import { LinkWithIcon } from '../LinkWithIcon';
```

Comment:
Although might be ok for now, but one thing we might want to think about is dynamic imports around these. If someone imports `CTA`, their build artifact will include all of these components, even if they may be using just one, for example. I don't think it's going to be a huge performance difference, but something to think about.